minio/minio
# turn-off checking for updates completely if MINIO_UPDATE=off

## Description
turn-off checking for updates completely if MINIO_UPDATE=off

## Motivation and Context
fixes #10723 

## How to test this PR?
Nothing much just check that we don't see requests to dl.min.io when MINIO_UPDATE=off

## Types of changes

## Checklist:
